spring-boot/spring-events/src/main/java/io/reflectoring/eventsdemo/EventsDemoApplication.java

```diff
@@ -0,0 +1,17 @@
+package io.reflectoring.eventsdemo;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import io.reflectoring.eventsdemo.listeners.SpringBuiltInEventsListener;
+
+@SpringBootApplication
+public class EventsDemoApplication {
+
+	public static void main(String[] args) {
+		SpringApplication springApplication = new SpringApplication(EventsDemoApplication.class);
+		springApplication.addListeners(new SpringBuiltInEventsListener());
```

Comment:
No, it won't be registered twice since I have not added @Component for it. 
The reason for adding it manually is, we need the listener to be registered before startup so that we can get access for ApplicationStartingEvent and some other events, which will be fired at start. 
So, to demonstrate all the built-in events and to explain manual registering of listener for these kind of cases, I have given that example. I have explained the same in article as well.